LICENSE

```diff
@@ -49,5 +49,3 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
-
-See COPYRIGHT-debops for additional copyrights.
```

Comment:
Then also add the suggested text https://github.com/linux-system-roles/logging/pull/142#discussion_r440344875 in the suggested place near the beginning of the file `LICENSE`

Reply:
Sorry, maybe I did not fully understand what to do... Is this change on LICENSE ok?
```
diff --git a/LICENSE b/LICENSE
index 2a1e447..2de853b 100644
--- a/LICENSE
+++ b/LICENSE
@@ -1,9 +1,17 @@
-Some files are licensed under GPL-3.0-only.  These files have the text
+Some files are licensed under Copyright (C) 2015-2020 Maciej Delmanowski
+drybjed@gmail.com, Copyright (C) 2015-2020 DebOps https://debops.org/ and
+GPL-3.0-only.
+
+These files have the text
 
     SPDX-License-Identifier: GPL-3.0-only
 
 in the file header.
 
+See the file COPYRIGHT in https://debops.org/ for Copyright (C) 2015-2020
+Maciej Delmanowski drybjed@gmail.com, Copyright (C) 2015-2020 DebOps
+https://debops.org/.
+
 All other files, unless noted, are licensed under the MIT license.
 
 = GPL-3.0-only license text

```

Reply:
Not exactly.  The license is `GPL-3.0-only`.  The files are licensed under `GPL-3.0-only`.  The files are Copyright (C) 2015-2020 Maciej Delmanowski drybjed@gmail.com and Copyright (C) 2015-2020 DebOps https://debops.org/.  It is not correct to say "licensed under Copyright".

Try this:

"Some files are licensed under GPL-3.0-only.  These files have the text

     SPDX-License-Identifier: GPL-3.0-only
 
in the file header.  The files with that header are Copyright (C) 2015-2020 Maciej Delmanowski drybjed@gmail.com and Copyright (C) 2015-2020 DebOps https://debops.org/.

See the file `COPYING` for the full text of the GPL-3.0-only license."

Reply:
Note that this assumes that all of the files used from debops, and *only* those files, are both licensed under GPL-3.0-only, and Copyright (C) 2015-2020 Maciej Delmanowski drybjed@gmail.com and Copyright (C) 2015-2020 DebOps https://debops.org/.  It also assumes all other files are files that were created under Red Hat copyright by us, and are licensed under the MIT license.  If this ever changes in the future, e.g. if we create some new files that will be copyright Red Hat, but for some reason we decide to use GPLv3 for those files, we will have to change this text.  But for now, I think it is ok.